app/assets/javascripts/thredded/components/topics.es6

```diff
@@ -28,8 +28,8 @@
     }
   }
 
-  $(function() {
-    var $nodes = $(COMPONENT_SELECTOR);
+  window.Thredded.onPageLoad(() => {
```

Comment:
Does this mean we need to do this for every normally document ready function? because of how turbolinks interacts with jquery... Ugh. But I guess if turbolinks is important to users we need to support somehow. It's just this seems quite a big shift

Maybe worth noting this somewhere (because we might forget, and wonder why we can't just do jquery document ready)